# maps feedback topics (Float64) to TF joints
std_msgs/String[] tf_joints
std_msgs/String[] feedback_joints
